src/pages/index.js

```diff
@@ -93,3 +208,7 @@ profileAddBtn.addEventListener("click", () => {
   profileAddPopup.open();
   addFormValidator.toggleButtonState();
```

Comment:
**Might be better**

You could make a special method `resetValidation` for clearing errors and controlling the submit button.
It could look something like this:

```js

resetValidation() {
      this._toggleButtonState(); <== controlling the submit button ==

      this._inputList.forEach((inputElement) => {
        this._hideError(inputElement) <== clearing errors ==
      });

    }



```

And you can call it in `index.js` when you click on open buttons